e2e/cypress/tests/integration/collapsed_reply_threads/global_threads_spec.js

```diff
@@ -320,4 +329,88 @@ describe('Collapsed Reply Threads', () => {
             cy.apiDeletePost(rootPost.id);
         });
     });
+
+    it('MM-T4448 CRT - L16 - Cancel “Mark all as unread” modal', () => {
```

Comment:
```suggestion
    it('MM-T4448_2 CRT - L16 - Cancel “Mark all as unread” modal', () => {
```

Also please update line 267 with updated test case id `it('MM-T4448_1 CRT - L16 - Use “Mark...`

Reply:
@jgilliam17 I updated it!